client/my-sites/plans/current-plan/feature-not-included-card/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Card, Gridicon } from '@automattic/components';
+
+import './style.scss';
+
+interface Props {
+	illustration: string;
+	text: string;
+	title: string;
+}
+
+const FeatureNotIncludedCard: React.FunctionComponent< Props > = ( props ) => {
+	const { illustration, title, text } = props;
+
+	return (
+		<Card className="feature-not-included-card__card">
+			<img className="feature-not-included-card__illustration" alt={ title } src={ illustration } />
+			<div>
+				<Gridicon className="feature-not-included-card__cross" icon="cross" size={ 24 } />
```

Comment:
I still prefer a `+` here, but that we can iterate on after discussing with the designers.

Reply:
Resolving this as we wait for the P2 response (and handle it in a new PR) 
peapX7-1yI-p2#comment-1737